src/identities/providers/publickey.js

```diff
@@ -27,7 +27,7 @@ const verifyIdentity = identity => {
  * identity provider function.
  * @private
  */
-const PublicKeyIdentityProvider = ({ keystore }) => {
+const PublicKeyIdentityProvider = ({ keystore }) => () => {
```

Comment:
Wondering if we should make the second resented function async? That’s how stores and ACs are and may be useful for 3rd party devs to do async io inside the closure.

Reply:
If so, need to add `await` to https://github.com/orbitdb/orbitdb/pull/1090/files#diff-8c445a0a80e9ec75a67dd06dcd352e8bfac45b3a9c67d662a65400c253141733R76